Automattic/newspack-plugin
# feat(ia): audience subscriptions boilerplate

### All Submissions:


### Changes proposed in this Pull Request:

Add New IA for Audience > Subscription wizard.

### How to test the changes in this Pull Request:

1. Checkout and build assets `git checkout origin/feat/ia-audience-subscriptions && npm run build`
2. Navigate to _/wp-admin/admin.php?page=newspack-audience-subscriptions_
3. Observe Configuration tab and card load correctly.
![Screenshot 2024-12-12 at 08 57 56](https://github.com/user-attachments/assets/0378a40a-85fd-4638-bd59-a1affd2511fb)
4. Navigate to "Revenue" tab. Observe Revenue tab and card load correctly.
![Screenshot 2024-12-12 at 08 59 17](https://github.com/user-attachments/assets/ced4f526-774b-42ae-9089-393082e1e0af)


### Other information:
